draft-ietf-quic-transport.md

```diff
@@ -5018,6 +5039,27 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+The Retire Prior To field is a request for the peer to retire all connection IDs
+with a sequence number less than the specified value.  This includes the initial
+and preferred_address transport parameter connection IDs.  The peer SHOULD
+retire the corresponding connection IDs and send the corresponding
+RETIRE_CONNECTION_ID frames in a timely manner
```

Comment:
This feels duplicate to the first diff of this PR. Maybe we can move the sentence about the initial SPA connection ID up there?